src/binded.js

```diff
@@ -0,0 +1,6 @@
+const methodReducer = obj => (state, methodName) =>
+  Object.assign({}, state, { [methodName]: obj[methodName].bind(obj) });
```

Comment:
i'm not arguing, just need to prevent future pull-requests to not pass this check